UI/html/update-order.html

```diff
@@ -0,0 +1,110 @@
+<!doctype>
```

Comment:
Please, specify a specific doctype. Check out the different doctype options, my recommendation is html5. Also, apply this in every other instance where you have left out the specific doctype